pepito_alonajane/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pepito, Alona Jane Navarro</title>
+    <link href="style.css" rel="stylesheet"/>
+  </head>
+  <body>
+    <div class="box">
+      <h1 class="box-heading">Alona Jane Navarro Pepito </h1>
+        <p class="box-paragraph">Welcome to my website!</p>
+          <div class="container">
+           <div class="section">
+             <h2>My interests and hobbies</h2>
+                <ul>
+                  <li>I love to cook</li>
+                  <li>I love to watch movies</li>
+                  <li>I like to go to new places</li>
+                  <li>I love cats</li>
+                  <li>I love listening songs</li>
+                </ul>
+            </div> 
+         </div>
+      <div class="container">
+        <div class="section">
+         <h2>My works</h2>
+           <ol>
+             <li><a href="./a1.html">html 1</a></li>
+             <li><a href="./a2.html">html 2</a></li>
+             <li><a href="./a3.html">html 3</a></li>
+             <li><a href="./a4.html">html 4</a></li>
+             <li><a href="./a5.html">html 5</a></li>
```

Comment:
Your work html files seem to have an unimportant content and you can remove it instead, but if you can make your html files have a work content, much better, but if you can't just remove the html files and replace the codes into this,
`<li><a href="#>work 1</a></li>`
(This is an example)